selftests/unit/test_safeloader.py

```diff
@@ -392,6 +414,17 @@ def test_recursive_discovery(self):
                                    ('test_basic', {})]}
         self.assertEqual(expected, tests)
 
+    def test_recursive_discovery_unittest(self):
```

Comment:
There will be better to use `test_recursive_discovery_python_unittest`. I commented this in #3204, but maybe we didn't understand each other.

Reply:
Right, forgot that.